src/main.py

```diff
@@ -98,18 +98,23 @@
     # such as unsafe-inline or unsafe-eval is highly discouraged, as these
     # may lead to XSS attacks.
     'csp_policy': {
-        # https://developers.google.com/fonts/docs/technical_considerations
-        'font-src':    '\'self\' themes.googleusercontent.com '
-                       '*.gstatic.com',
+        # Fallback.
+        'default-src': '\'self\'',
+        # Disallow Flash, etc.
+        'object-src': '\'none\'',
+        # Google Analytics.
+        'script-src': '\'self\' https://www.google-analytics.com/analytics.js',
         # Maps, YouTube provide <iframe> based embedding at these URIs.
-        'frame-src':   '\'self\' www.google.com www.youtube.com',
-        # Assorted Google-hosted APIs.
-        'script-src':  '\'self\' *.googleanalytics.com *.google-analytics.com',
+        'child-src':  '\'self\' https://www.google.com https://www.youtube.com',
+        # Deprecated. Used for supporting browsers that use CSP 1.0 only.
+        'frame-src':  '\'self\' https://www.google.com https://www.youtube.com',
```

Comment:
Is there any public telemetry where we can evaluate the worth of keeping the deprecated clause?  (i.e. if it's below X%, maybe that's an acceptable cutoff)